packages/iTwinUI-react/package.json

```diff
@@ -35,7 +35,6 @@
     "clean:coverage": "rimraf coverage",
     "test": "jest",
     "test:watch": "jest --watch",
-    "createComponent": "node scripts/createComponent.js",
```

Comment:
not sure about this.. Maybe it should be possible to create component from both, root and itwinui-react package?
Even though now as storybook is moved onto separate folder, it might make less sense :/

Reply:
It's currently set up to make components from the root (even though the file itself is inside packages, i'll move to root). Do you think if we had two scripts (one for root and one for package) it would cause any issues? The scripts shouldn't interfere with any existing code, but you would run the command the same in either directory and have the same output.

Reply:
oh no no.. only single script.
I meant to add `createComponent` command in both `package.json` (root and package itself).
But it might make sense to have it only in root, so its fine.

Reply:
I left a comment below about how to achieve this. Then you can just leave this line as it was before.

Reply:
Ok added this in, thanks!

Reply:
You'll need to add `"createComponent": "node scripts/createComponent.js",` back in the file `packages/iTwinUI-react/package.json`.

Reply:
Got it